docs/resiliency/index.asciidoc

```diff
@@ -102,18 +110,30 @@ Indices stats and indices segments requests reach out to all nodes that have sha
 while the stats request arrives will make that part of the request fail and are just ignored in the overall stats result. {GIT}13719[#13719]
 
 [float]
-=== Jepsen Test Failures (STATUS: ONGOING)
+=== Document guarantees and handling of failure (STATUS: ONGOING)
```

Comment:
Maybe "Documentation of guarantees" so that "Document" can not be confused with the notion of an indexed document?